docs/api/algorand/check-transaction.mdx

```diff
@@ -1,16 +1,60 @@
 # CheckTransaction
```

Comment:
Also, please update the `SubmitTransaction` docs description to tell that you need to always check the status of transaction using `CheckTransaction`.